app/src/main/java/com/example/appmarvel/services/api/mapper/CharacterMapper.kt

```diff
@@ -0,0 +1,20 @@
+package com.example.appmarvel.services.api.mapper
+
+import com.example.appmarvel.entity.Character
+import com.example.appmarvel.services.response.DataResponse
+
+class CharacterMapper : BaseMapper<DataResponse, MutableList<Character>> {
```

Comment:
instead of using a class, this can be an object.. or better, an extension function

Reply:
Done